ra-optimisation/rao-api/src/main/java/com/farao_community/farao/rao_api/Rao.java

```diff
@@ -0,0 +1,197 @@
+
+        private final RaoProvider provider;
+
+        public Runner(RaoProvider provider) {
+            this.provider = Objects.requireNonNull(provider);
+        }
+
+        public CompletableFuture<RaoComputationResult> runAsync(Network network, Crac crac, String workingStateId, ComputationManager computationManager, RaoParameters parameters) {
+            Objects.requireNonNull(workingStateId);
+            Objects.requireNonNull(parameters);
+            return provider.run(network, crac, workingStateId, computationManager, parameters);
+        }
+
+        public CompletableFuture<RaoComputationResult> runAsync(Network network, Crac crac, ComputationManager computationManager, RaoParameters parameters) {
+            return runAsync(network, crac, network.getVariantManager().getWorkingVariantId(), computationManager, parameters);
+        }
+
+        public CompletableFuture<RaoComputationResult> runAsync(Network network, Crac crac, RaoParameters parameters) {
+            return runAsync(network, crac, LocalComputationManager.getDefault(), parameters);
+        }
+
+        public CompletableFuture<RaoComputationResult> runAsync(Network network, Crac crac) {
+            return runAsync(network, crac, RaoParameters.load());
+        }
+
+        public RaoComputationResult run(Network network, Crac crac, String workingStateId, ComputationManager computationManager, RaoParameters parameters) {
+            Objects.requireNonNull(workingStateId);
+            Objects.requireNonNull(parameters);
+            return provider.run(network, crac, workingStateId, computationManager, parameters).join();
+        }
+
+        public RaoComputationResult run(Network network, Crac crac, ComputationManager computationManager, RaoParameters parameters) {
+            return run(network, crac, network.getVariantManager().getWorkingVariantId(), computationManager, parameters);
+        }
+
+        public RaoComputationResult run(Network network, Crac crac, RaoParameters parameters) {
+            return run(network, crac, LocalComputationManager.getDefault(), parameters);
+        }
+
+        public RaoComputationResult run(Network network, Crac cracFile) {
+            return run(network, cracFile, RaoParameters.load());
+        }
+
+        @Override
+        public String getName() {
+            return provider.getName();
+        }
+
+        @Override
+        public String getVersion() {
+            return provider.getVersion();
+        }
+    }
+
+    /**
+     * Get a runner for optimising RA named {@code name}. In the case of a null {@code name}, default
+     * implementation is used.
+     *
+     * @param name name of the RAO implementation, null if we want to use default one
+     * @return a runner for RAO implementation named {@code name}
+     */
+    public static Runner find(String name) {
+        return find(name, PROVIDERS_SUPPLIERS.get(), PlatformConfig.defaultConfig());
```

Comment:
name is not a very good variable name I would use raoName which is more explicit

Reply:
As soon as it is described in javadoc, I think a short name like this is ok, especially in a small method like this